Artur_Badalyan/Day2/gameScript

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#Game
+
+number=$((1 + $RANDOM % 20))
+
+for i in {1..3}
+do
+    echo "Write number for Game\n"    
+    read input    
+    if [ $input -eq $number ]
+    then
+        printf "Yes you find number\n"
+    elif [ $input -lt $number ]
+    then        
+        printf "Your input number small in memorable number\n"
+    elif [ $input -gt $number ]
+    then        
+        printf "Your input number great in memorable number\n"        
+    fi
+done
+
+printf "Memorable number is $number\n Try again!!\n"
+
```

Comment:
- need to inform the user about the range of numbers
- need to finish the game if the number is correct (e.g. during the first attempt)
- no check for input values